.travis.yml

```diff
@@ -34,6 +34,15 @@ addons:
     # https://httpie.org is used by the docker-compose test below
     - httpie
 
+before_script:
```

Comment:
FYI (minor) this will conflict with #846, maybe we can get that one merged first?